lib/graphql/object/field.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+# test_via: ../object.rb
+
+module GraphQL
+  class Object
+    class Field
+      def initialize(name, return_type_expr, desc = nil, null: false, deprecation_reason: nil, method: nil, &args_block)
```

Comment:
I would suggest making `null:` a required argument with no default; we've found internally that otherwise people simply forget to think about it more often than not.

Reply:
:+1:

Reply:
@eapache @rmosolgo agreed x1000, especially for Public APIs (because when it's deployed it's already too late to change 🙈)

It's pretty nice to make gem users to think about `null: true` vs `null: false`